web/dart/session/movable.dart

```diff
@@ -316,6 +316,21 @@ class Movable extends InstanceComponent
     applyConditions(List<int>.from(json['conds'] ?? []));
     onPrefabUpdate();
   }
+
+  void ping() {
+    this.styleSelected = true;
+    Future.delayed(const Duration(seconds: 10), () {
+      this.styleSelected = false;
+    });
+  }
+
+  void goTo() {
+    this.styleSelected = true;
+    board.animateTransformToToken(this);
+    Future.delayed(const Duration(seconds: 10), () {
+      this.styleSelected = false;
+    });
+  }
```

Comment:
If a player had previously selected this specific token/movable, the styling would disappear after 10 seconds. Instead, it'd be more stable to add a different CSS class (e.g. "pinged") to the element for a short duration.